_episodes/07-docker-image-examples.md

```diff
@@ -28,13 +28,36 @@ In this [GitHub Actions example](../_extras/e01-github-actions), you can learn m
 continuous integration in the clound and how you can use container images with GitHub to
 automate some of your workflow.
 
-<!--- Placeholder for 
+<!--- Placeholder for
 ## Geospatial Example
 
 Ask @mkuzak to make a PR to add extra for <https://github.com/escience-academy/docker-gdal-demo>
 
 -->
 
+## On an HPC Cluster
```

Comment:
Maybe add words to make this heading self-contained?

Reply:
can you give an example of what you mean?